src/cases/containers/case-share/case-share.component.ts

```diff
@@ -43,7 +43,7 @@ export class CaseShareComponent implements OnInit {
       // call api to retrieve users in the same organisation
       this.store.dispatch(new LoadUserFromOrgForCase());
     }
-    this.removeUserFromCaseToggleOn$ = this.featureToggleService.getValue('remove-user-from-case', false);
+    this.removeUserFromCaseToggleOn$ = this.featureToggleService.getValue('remove-user-from-case-mc', false);
```

Comment:
This string appears in at least two places and seems to have recently changed. It would be sensible to make this a constant somewhere to aid maintainability.

Reply:
moved to a constant